oak_runtime/src/node/http/tests.rs

```diff
@@ -45,6 +46,58 @@ impl crate::node::Node for OakNodeForTest {
         let invocation_receiver = Receiver::<HttpInvocation>::new(ReadHandle { handle });
 
         while let Ok(invocation) = invocation_receiver.receive(&runtime) {
+            // Create a request-specific EchoNode, with a privilege that allows the node to
+            // declassify the request.
+            let privilege = NodePrivilege {
+                can_declassify_confidentiality_tags: hashset! {tls_endpoint_tag("localhost")},
```

Comment:
This does not seem to create a request-specific node, or better, it seems to create the same exact node for each request, regardless of the label of the request, right? It would make more sense to actually create it based on the label of the request. Which also applies to line 71 below, in which it looks like the newly created node is actually being created as public, instead of matching the request label, so that should also be changed I think.

Reply:
Are you suggesting that the node should not have the privilege, but should have the same label as the request label? 
In that case, it will be able to read the request but not write the response. Should the response channel be created with the request label too?